src/main/kotlin/com/strumenta/kolasu/model/Traversing.kt

```diff
@@ -0,0 +1,124 @@
+package com.strumenta.kolasu.model
+
+import java.util.*
+
+/**
+ * Some Kotlinization of Deques used as a stack.
+ */
+typealias Stack<T> = Deque<T>
```

Comment:
Nice touch

Reply:
Not sure how nice - it has the same name as the Java Stack class, and it exposes a bunch of non-stack methods. It shows intention better though. We can always throw it out again.